packages/arb-token-bridge-ui/src/util/withdrawals/fetchTokenWithdrawalsFromEventLogs.ts

```diff
@@ -1,14 +1,7 @@
 import { Provider, BlockTag } from '@ethersproject/providers'
-import { Erc20Bridger, EventArgs } from '@arbitrum/sdk'
+import { Erc20Bridger } from '@arbitrum/sdk'
 import { WithdrawalInitiatedEvent } from '@arbitrum/sdk/dist/lib/abi/L2ArbitrumGateway'
-
-function dedupeEvents(
-  events: (EventArgs<WithdrawalInitiatedEvent> & {
-    txHash: string
-  })[]
-) {
-  return [...new Map(events.map(item => [item.txHash, item])).values()]
-}
+import { dedupeEvents } from '../deposits/helpers'
```

Comment:
Withdrawals shouldn't really import from deposits helpers, maybe we can move it to `/TransactionHistory/helpers` instead?